aries-site/src/pages/tokens/all-design-tokens.js

```diff
@@ -0,0 +1,347 @@
+/* eslint-disable react/prop-types */
```

Comment:
I wasn't sure where to put this comment so adding it here.... is there a way that we can make the search be across the family (primitive, semantic and component) instead of specific to the lower lever (semantic/color)? It feels like it might be easier to search on that level once we are used to the naming conventions. For example: I go into semantic and search for color background instead of having to navigate to color to search the specific token.
Just an opinion though.

Reply:
Good comment, we can look into how to refine the search experience. Going to put this on hold for now with the other release prep I'm doing. If needed can file as separate enhancement.

Reply:
Sounds good to me!